docs/source/guides/development/development.rst

```diff
@@ -0,0 +1,12 @@
+=======================
+Application Development
+=======================
+
+The NuCypher Network provides cryptographic infrastructure for privacy-preserving applications.
```

Comment:
Looking forward to adding much more stuff here!!

Reply:
👍